pkgs/top-level/python-packages.nix

```diff
@@ -2737,6 +2737,8 @@ in {
 
   python2-pythondialog = callPackage ../development/python-modules/python2-pythondialog { };
 
+  strict-rfc3339 = callPackage ../development/python-modules/strict-rfc3339 { };
```

Comment:
Can you sort this with another attribute that in alphabetical  succession?
It's not really alphabetical but that still helps to reduce merge conflicts.

Reply:
Done